databases/week2/create_db_lesson2_part4.sql

```diff
@@ -34,12 +34,12 @@ CREATE TABLE ´ bike ´(
   ´ id ´ int(10) UNSIGNED NOT NULL AUTO_INCREMENT,
   ´ made_in ´ VARCHAR(255) NOT NULL,
   ´ quality ´ VARCHAR(255),
-  ´ date_of_manufacture ´ DATETIME,
+  ´ date ´ DATETIME,

```

Comment:
i meant the data type could be DATE, i.e, instead of DATETIME you could have DATE, the name of the column was very good. sorry i wasn't clear :/